client/src/redux/createStore.js

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
+import reducers from './reducers';
+
+const store = createStore( reducers );
```

Comment:
It would be nice if we could also inspect Redux using devtools. Can we modify the store using this line?

```
const store = createStore(reducers, window.__REDUX_DEVTOOLS_EXTENSION__ &&
  window.__REDUX_DEVTOOLS_EXTENSION__());
```